kratos/tests/cpp_tests/geometries/test_quadrature_point_geometry.cpp

```diff
@@ -104,5 +133,44 @@ namespace Kratos {
             KRATOS_CHECK_EQUAL(p_this_quadrature_point->ShapeFunctionsValues().size2(), 3);
         }
 
+        KRATOS_TEST_CASE_IN_SUITE(QuadraturePointGeometry2dCopyConstructor, KratosCoreFastSuite)
+        {
+            auto p_this_quadrature_point = GenerateQuadraturePointGeometry2();
```

Comment:
here the `auto` makes it a bit harder to read, one has to check the function to see what the type is